front/components/poke/workspace/table.tsx

```diff
@@ -25,15 +57,69 @@ export function WorkspaceInfoTable({
         <PokeTable>
           <PokeTableBody>
             <PokeTableRow>
-              <PokeTableCell>Workspace creation</PokeTableCell>
+              <PokeTableCell>Id</PokeTableCell>
+              <PokeTableCell>
+                {owner.id}
+                &nbsp;
+                <Button
+                  size="xs"
+                  variant="secondary"
+                  onClick={useCallback(
+                    async (e: any) => {
+                      e.preventDefault();
+                      await copyToClipboardId(
+                        new ClipboardItem({
+                          "text/plain": new Blob([`${owner.id}`], {
+                            type: "text/plain",
+                          }),
+                        })
+                      );
+                    },
+                    [copyToClipboardId, owner.id]
+                  )}
+                  label="Copy"
+                  labelVisible={false}
+                  icon={isCopiedId ? ClipboardCheckIcon : ClipboardIcon}
+                />
+              </PokeTableCell>
+            </PokeTableRow>
+            <PokeTableRow>
+              <PokeTableCell>sId</PokeTableCell>
+              <PokeTableCell>
+                {owner.sId}
+                &nbsp;
+                <Button
+                  size="xs"
+                  variant="secondary"
+                  onClick={useCallback(
+                    async (e: any) => {
+                      e.preventDefault();
+                      await copyToClipboardSid(
+                        new ClipboardItem({
+                          "text/plain": new Blob([owner.sId], {
+                            type: "text/plain",
+                          }),
+                        })
+                      );
+                    },
+                    [copyToClipboardSid, owner.sId]
+                  )}
+                  label="Copy"
+                  labelVisible={false}
+                  icon={isCopiedSid ? ClipboardCheckIcon : ClipboardIcon}
+                />
+              </PokeTableCell>
+            </PokeTableRow>
+            <PokeTableRow>
+              <PokeTableCell>Creation</PokeTableCell>
               <PokeTableCell>{worspaceCreationDay}</PokeTableCell>
             </PokeTableRow>
             <PokeTableRow>
-              <PokeTableCell>SSO Enforced</PokeTableCell>
+              <PokeTableCell>SSO</PokeTableCell>
```

Comment:
Why removing enforced? SSO can be enabled but not enforced.

Reply:
I wanted to shorten but I didn't know that it could be enabled and not enforced, fixing.